### PR TITLE
Update Kubernetes/Openmetrics instructions and samples

### PR DESCRIPTION
**Pull request by JacksonDavenport**
### What does this PR do?

Update the instructions for Openmetrics V2 with more sample metrics. Includes updates to the new linked image as well as `prometheus.yaml` sample file. 

### Motivation

Support cases

### Preview

Preview Build:
https://docs-staging.datadoghq.com/jack.davenport/k8s_prometheus/agent/kubernetes/prometheus/

Current: 
https://docs.datadoghq.com/agent/kubernetes/prometheus/

### Additional Notes

Will create changes for the Docker/Prometheus page in a separate PR

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/kubernetes/prometheus.md [hunk 1]
```diff
@@ -45,91 +45,90 @@ For an explanation of how Prometheus and OpenMetrics metrics map to Datadog metr
...
+| `<PROMETHEUS_ENDPOINT>`                  | URL path for the metrics served by the container, in Prometheus format.                            |
+| `<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>` | Set namespace to be prefixed to every metric when viewed in Datadog.                               |
+| `<METRIC_TO_FETCH>`                      | Prometheus metrics key to be fetched from the Prometheus endpoint.                                 |
+| `<NEW_METRIC_NAME>`                      | Transforms the `<METRIC_TO_FETCH>` metric key to `<NEW_METRIC_NAME>` in Datadog.                   |
 
-For a full list of available parameters for instances, including `namespace` and `metrics`, see the table in the [Prometheus host collection guide][9].
 
+The `metrics` configuration is list of metrics to retrieve as custom metrics. Include each metric to fetch and the desired metric name in Datadog as `{"<METRIC_TO_FETCH>":"<NEW_METRIC_NAME>"}` pairs. You can alternatively provide a list of metric names strings, interpreted as regular expressions, to bring the desired metrics with their current names. **Note:** Regular expressions can potentially send a lot of custom metrics.
```
Comment (jtappa) [inline content/en/agent/kubernetes/prometheus.md:83]:
```suggestion
The `metrics` configuration is a list of metrics to retrieve as custom metrics. Include each metric to fetch and the desired metric name in Datadog as key value pairs, for example, `{"<METRIC_TO_FETCH>":"<NEW_METRIC_NAME>"}`. You can alternatively provide a list of metric names strings, interpreted as regular expressions, to bring the desired metrics with their current names. **Note:** Regular expressions can potentially send a lot of custom metrics.
```

File: content/en/agent/kubernetes/prometheus.md [hunk 2]
```diff
@@ -45,91 +45,90 @@ For an explanation of how Prometheus and OpenMetrics metrics map to Datadog metr
...
-| `<METRIC_TO_FETCH>`                      | Prometheus metrics key to be fetched from the Prometheus endpoint.                                                                                                                                             |
-| `<NEW_METRIC_NAME>`                      | Optional parameter which, if set, transforms the `<METRIC_TO_FETCH>` metric key to `<NEW_METRIC_NAME>` in Datadog. If you choose not to use this option, pass a list of strings rather than `key:value` pairs. |
+| Placeholder                              | Description                                                                                        |
+|------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `<CONTAINER_IDENTIFIER>`                 | The identifier used in the `annotations` needs to match the container `name` exposing the metrics. |
```
Comment (jtappa) [inline content/en/agent/kubernetes/prometheus.md:76]:
```suggestion
| `<CONTAINER_IDENTIFIER>`                 | The identifier used in the `annotations` must match the container `name` exposing the metrics. |
```

File: content/en/agent/kubernetes/prometheus.md [hunk 3]
```diff
@@ -45,91 +45,90 @@ For an explanation of how Prometheus and OpenMetrics metrics map to Datadog metr
 
 ### Configuration
 
-Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your OpenMetrics/Prometheus **pod**:
+Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your **pod** exposing the OpenMetrics/Prometheus metrics:
```
Comment (jtappa) [inline content/en/agent/kubernetes/prometheus.md:48]:
```suggestion
Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your **pod** and exposing the OpenMetrics/Prometheus metrics:
```

**Review by jtappa — CHANGES_REQUESTED** [review]

File: content/en/agent/kubernetes/prometheus.md (hunk 3 above)
Comment (JacksonDavenport) [reply content/en/agent/kubernetes/prometheus.md:48]: Skipping this one as the pod/container itself is already exposing these metrics. Our Datadog Autodiscovery annotations are the configurations to go and collect those exposed metrics, not necessarily the configuration to expose those.

**Review by jtappa — APPROVED** [review]

**Pull request merged**